kratos/processes/variational_distance_calculation_process.h

```diff
@@ -123,9 +123,11 @@ class VariationalDistanceCalculationProcess : public Process
         ModelPart& base_model_part,
         typename TLinearSolver::Pointer plinear_solver,
         unsigned int max_iterations = 10,
-        Flags Options = NOT_CALCULATE_EXACT_DISTANCES_TO_PLANE)
-        :mr_base_model_part(base_model_part),
-         mOptions(Options)
+        Flags Options = NOT_CALCULATE_EXACT_DISTANCES_TO_PLANE,
+        std::string aux_part_name = "RedistanceCalculationPart" )
```

Comment:
```suggestion
        std::string AuxPartName = "RedistanceCalculationPart" )
```
To be consistent with the style guide.